www/docs/en/dev/guide/platforms/android/webview.md

```diff
@@ -29,106 +29,73 @@ components can communicate with each other, see Application Plugins.
 If you're unfamiliar with Android, you should first familiarize
 yourself with the [Android Platform Guide](index.html) and have the latest Android
 SDK installed before you attempt the more unusual development option
-of embedding a WebView.  Starting with Cordova 1.9, the Android
-platform relies on a `CordovaWebView` component, which builds on a
-legacy `CordovaActivity` component that pre-dates the 1.9 release.
+of embedding a WebView.  Starting with Cordova 4.0, the Android
+platform relies on a `SystemWebView` component.
 
 1. To follow these instructions, make sure you have the latest Cordova
    distribution. Download it from
    [cordova.apache.org](http://cordova.apache.org) and unzip its
    Android package.
 
-1. Navigate to the Android package's `/framework` directory and run
-   `ant jar`. It creates the Cordova `.jar` file, formed as
-   `/framework/cordova-x.x.x.jar`.
+1. Create a project in Android Studio
 
-1. Copy the `.jar` file into the Android project's `/libs` directory.
+1. Copy the framework directory into your project's root directory and name it
+CordovaLib
+
+1. Add that project as a library project to your main project.
 
 1. Add the following to the application's `/res/xml/main.xml` file,
    with the `layout_height`, `layout_width` and `id` modified to suit
    the application:
 
-        <org.apache.cordova.CordovaWebView
+        <org.apache.cordova.SystemWebView
```

Comment:
It's org.apache.cordova.engine.SystemWebView, not org.apache.cordova.SystemWebView